client/src/components/import-component/index.jsx

```diff
@@ -0,0 +1,198 @@
+import { useState, useEffect, useRef } from "react";
+import {
+  Typography,
+  Stack,
+  Card,
+  CardHeader,
+  CardContent,
+  Select,
+  MenuItem,
+  FormHelperText,
+  Button,
+  FormControl,
+  InputLabel,
+  Paper,
+  Grid,
+  IconButton,
+  LinearProgress,
+  Box,
+} from "@mui/material";
+import CloseIcon from "@mui/icons-material/Close";
+import { Form, Header, Upload, UploadIcon, Cancel } from "./index.styles";
+
+function importComponent() {
+  const [menu, setMenu] = useState("");
+  const [files, setFiles] = useState([]);
+  const [progress, setProgress] = useState(0);
+
+  useEffect(() => {
+    const timer = setInterval(() => {
+      setProgress((oldProgress) => {
+        const diff = Math.random() * 10;
+        return Math.min(oldProgress + diff, 100);
+      });
+    }, 500);
+
+    return () => {
+      clearInterval(timer);
+    };
+  }, []);
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    // Create a new FormData object
+    const formData = new FormData();
+    // Add the file(s) to the FormData object
+    formData.append("files", files);
+
```

Comment:
Create a method for parsing the csvs that returns a list of objects -> one object for each row of the csv